_data/students.yml

```diff
@@ -92,3 +92,9 @@
   facebook: https://facebook.com/cuteaditya1
   email: blusinghaditya@gmail.com
   bio: Programmer & Web Designer, Second year GCI!
+  
+- name: Baidyanath
+  github: Baidya99
+  image: Baidya.jpg
```

Comment:
Change your image link. You have added your image in `images/` folder in https://github.com/fossasia/gci16.fossasia.org/pull/35 and please do the photo upload and changes in `students.yml` in the same pull request.